holoviews/core/overlay.py

```diff
@@ -230,6 +263,25 @@ def ddims(self):
     def shape(self):
         raise NotImplementedError
 
+    # Deprecated methods
+
+    def collapse(self, function):
+        "Deprecated method to collapse layers in the Overlay."
+        if config.future_deprecations:
+            self.warning('Overlay.collapse has been deprecated, to'
```

Comment:
You would find many of the other warnings searching for 'is deprecated' but this one is 'has been deprecated'.

Reply:
```suggestion
            self.warning('Overlay.collapse is deprecated, to'
```